<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rezervacija</class>
 <widget class="QDialog" name="rezervacija">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Rezervacija&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0" colspan="3">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>karta u jednosmjernom pravcu</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>slobodni letovi:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QComboBox" name="let"/>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="brtel"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>prezime:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="3">
     <widget class="QLabel" name="status">
      <property name="text">
       <string>status:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="prezime"/>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>br. telefona:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>izbor klase:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>povratak:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>rezervisi</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="ime"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ime:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>datum leta:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QComboBox" name="klasa"/>
    </item>
    <item row="5" column="2">
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QDateTimeEdit" name="dateTimeEdit_2">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked()</signal>
   <receiver>dateTimeEdit_2</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked()</signal>
   <receiver>label_7</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
